pandas/compat/__init__.py

```diff
@@ -76,8 +120,18 @@ def _import_lzma():
 
 
 def _get_lzma_file(lzma):
-    """Returns the lzma method LZMAFile when the module was correctly imported.
-    Otherwise, raises a RuntimeError.
+    """
+    Attempting to get the lzma.LZMAFile method.
```

Comment:
The old doc string was actually wrong, and it should say "... get the lzma.LZMAFile class". Can you change that, else looks good.